test/verify/check-storage-vdo

```diff
@@ -74,12 +74,24 @@ class TestStorage(StorageCase):
 
         # Make a filesystem on it
 
-        m.execute("mkfs.xfs -f -K /dev/mapper/vdo0")
-        m.execute("mkdir /run/data && mount /dev/mapper/vdo0 /run/data")
+        self.content_row_wait_in_col(1, 1, "Unrecognized Data")
+        self.content_tab_action(1, 1, "Format")
+        self.dialog({"type": "xfs",
+                     "name": "FILESYSTEM",
+                     "mounting": "custom",
+                     "mount_point": "/run/data"})
         self.content_row_wait_in_col(1, 1, "xfs File System")
+        # _netdev etc should have been prefilled
+        self.content_tab_wait_in_info(1, 1, "Mount Options", "_netdev")
+        self.content_tab_wait_in_info(1, 1, "Mount Options", "x-systemd.device-timeout=0")
+        self.content_tab_wait_in_info(1, 1, "Mount Options", "x-systemd.requires=vdo.service")
+        self.content_tab_action(1, 1, "Mount")
         self.content_tab_wait_in_info(1, 1, "Mounted At", "/run/data")
         self.content_tab_wait_in_info(1, 1, "Used", "of 4.99 GiB", "of 5.0 GiB")
 
+        # XXX - Clicking "Mount" above triggers this React warning.
+        self.allow_browser_errors("Warning: Cannot update during an existing state transition")
```

Comment:
FTR: This gets fixed in #10823, and not a (direct) regression from this PR, so fine for now.